src/app/parcels/edit/EditParcelForm.tsx

```diff
@@ -111,19 +112,18 @@ const EditParcelForm = ({ parcelId }: EditParcelFormProps): React.ReactElement =
             );
             setPackingSlotsIsShown(parcelData.packing_slot?.is_shown);
             setCollectionCentreIsShown(parcelData.collection_centre?.is_shown === true);
-
             setIsLoading(false);
         })();
     }, [parcelId]);
 
     const initialFormErrors: ParcelErrors = {
         voucherNumber: Errors.none,
         packingDate: Errors.none,
-        packingSlot: Errors.none,
+        packingSlot: packingSlotIsShown ? Errors.none : Errors.invalidPackingSlot,
         shippingMethod: Errors.none,
         collectionDate: Errors.none,
         collectionTime: Errors.none,
-        collectionCentre: Errors.none,
+        collectionCentre: collectionCentreIsShown ? Errors.none : Errors.invalidCollectionCentre,
```

Comment:
Think I'm a bit confused by errors in general, do you mind explaining what is happening here? Why are most of them none, and why would a packing slot or collection centre show an error - is it if that type was deleted and the box in the edit is now blank? Is it affected by whether it's a collection or a delivery (i.e. the extra section is expanded)?

Think an example would be helpful to understand when these would appear

Reply:
Yep: if the parcel has an AM slot for example and an admin deletes (or hides) that slot, when the form is opened the packing slot will be shown as blank and an error message (saying this packing slot no longer exists.. or something like that) will show. This should be equivalent to how it was before, it was originally done via useStates that weren't necessary methinks.

Reply:
They are mostly none as a parcel that you're editing shouldn't have any errors, as it wouldn't have allowed you to created it in the first place, just those edge cases that could cause it. I don't think collection or delivery should affect things - though the invalidCollectionCentre would only show if the parcel was a collection & the collection centre has subsequently been hidden/removed